public/js/controllers/index.js

```diff
@@ -57,8 +61,38 @@ angular.module('mean.system')
         };
 
         $scope.playAsGuest = () => {
-          game.joinGame();
-          $location.path('/app');
+          if ($scope.region === undefined || $scope.region === '') {
+            $scope.error = 'Select a region first!';
+          } else {
+            $('#close').click();
+            // $location.path('/app');
```

Comment:
Why is this line commented?
Seems this would likely not enable users to play as guests